src/components/NewTask.tsx

```diff
@@ -1,7 +1,19 @@
-import React, { useState, useContext } from 'react'
+import React, { useState, useContext, ChangeEvent } from 'react'
+import styled from 'styled-components'
+import {
+  Button,
+  Box,
+  Flex,
+  Grid,
+  Heading,
```

Comment:
```suggestion
```
We tend not to use Headings in the studio context. I've replaced them with Text as h2s (understandable if this doesn't make sense with your styling, just trying to stay consistent with our own packages!)